File: 백준/Bronze/10810. 공 넣기/공 넣기.py
import sys
from collections import deque
input = sys.stdin.readline
N, M = map(int, input().strip().split())
balls = [0 for _ in range(N)]

for _ in range(M):
    i, j, k = map(int, input().strip().split()) 
    for idx in range(i-1, j):
        balls[idx] = k

print(' '.join(map(str, balls)).strip())

File: 백준/Bronze/10810. 공 넣기/공 넣기.js
const input = require('fs')
  .readFileSync('/dev/stdin')
  .toString()
  .trim()
  .split('\n');

const [n, m] = input[0].split(' ').map((v) => +v);
const balls = new Array(n).fill(0);

for (let idx = 0; idx <= m; idx++) {
  let [i, j, k] = input[idx].split(' ').map((v) => +v);
  for (i; i <= j; i++) {
    balls[i - 1] = k;
  }
}
console.log(balls.join(" "));